### DISCUSSION TITLE
Is there a reason we haven't fixed the unmaintained advisorsies CI errors?

### DISCUSSION THREAD

**Discussion by dhuseby** [libp2p/rust-libp2p#4342] [General]
Getting CI errors here: https://github.com/libp2p/rust-libp2p/actions/runs/5897783308/job/15997906539?pr=4335#step:4:266

Issues #3136 and #3137 were opened back in November of 2022 and remain open. What's the story with these?

**Comment by thomaseizinger** [comment]
> Getting CI errors here

What are you pointing to is a warning, not an error.

> Issues #3136 and #3137 were opened back in November of 2022 and remain open. What's the story with these?

Thanks for the pointer! I looked into them and as expected, they come in via transitive dependencies, `webrtc-rs` to be precise. There is little we can do about those unless `webrtc-rs` updates their dependencies.

Our webrtc transport is in alpha state for various reasons (little experience with is performance and stability and overall low confidence in the library, see https://github.com/libp2p/rust-libp2p/issues/3659 for details). Thus, I'd say fixing those is low priority.

**Reply by thomaseizinger** [reply]
Btw, the actual error in that CI job has been addressed in https://github.com/libp2p/rust-libp2p/pull/4337.

**Comment by dhuseby** [comment]
Thanks for the explanation.
